### ISSUE TITLE
[Build] Unstable build on Apache CI

### ISSUE THREAD

**Issue by Arsnael** [linagora/james-project#4919]
More and more builds are failing lately. 

Mainly the cause seems linked to test containers failing to start correctly. I think it's maybe more proper to testcontainers itself or/and maybe some issues with the CI, as I don't only see Cassandra container failing now but also tika, opensearch, pulsar, ...

We should try to understand what's going on and maybe find ways to start our testing containers in a more robust way.

**Comment by Arsnael** [comment]
Can also disable unrelated tests to postgres on the postgres branch during dev

**Comment by vttranlina** [comment]
The maven test on only postgres modules is easier than disable unrelated tests to postgres
We can modify the Jenkinsfile with maven `-pl` argument, 
Commit https://github.com/apache/james-project/pull/1770/commits/2af723e73f917e332fe106dca21066aa2222d326

Note: in the future when we add more modules, we need also to add new module to `POSTGRES_MODULES`

**Comment by Arsnael** [comment]
I think better too. Thanks @vttranlina

**Comment by Arsnael** [comment]
https://github.com/apache/james-project/pull/1781
Not test containers related but other issues I saw :)

**Comment by Arsnael** [comment]
Just spotted runs failing because there was no docker environment... it seems some new VMs have spawned, like here in the logs:
```
Running on EC2 (Jenkins Ephemeral Node User-CLI) - t3.xlarge (i-07cdadd9591abfa17)
```

I will check more attentively and might fire later an issue on the apache infra jira cause I'm thinking sometimes our builds are failing because of the infra likely.

**Comment by Arsnael** [comment]
https://issues.apache.org/jira/browse/INFRA-25053

Ticket already exists :) I dropped a message to let infra know we experience that issue too.

Honestly I saw other tickets with some nodes needing to be cleaned out (out of space), etc... 

The builds look overall a bit more stable IMO these last few days. I think that Apache CI is having troubles atm and is working on solving them. Let's wait a bit longer to see.

If somebody else though wants to take a look and maybe enhance the build stability on other aspects, feel free :)
